model/product/product-controller.js

```diff
@@ -197,6 +232,47 @@ class ProductController extends Controller {
     }
   } // END : addPosition
 
+    findByAssociation(req, res, next){
+	const resCheck = checkArrayProduct(req.query);
+	
+	if (resCheck.code !== 200) {
+	    res.status(resCheck.code).send(resCheck.message);
+	    return;
+	}
+
+	return productFacade
+	    .findProductsByAssociation(req.query.eans)
+	    .then((collection) => {
+		if (collection === null || collection.length === 0) {
+		    res.status(404).send('No product associated with those products');
```

Comment:
Not sure we should return a `404` error in such a case, as there is nothing wrong with the incoming request sent by the app. There is just no associated product so far, but we cannot blame the app for that.